src/Raven.Server/ServerWide/ShardingStore.cs

```diff
@@ -103,6 +103,7 @@ public void ClearPublishedUrls()
                 UseHttpDecompression = _serverStore.Configuration.Sharding.ShardExecutorUseHttpDecompression,
                 GlobalHttpClientTimeout = _serverStore.Configuration.Sharding.OrchestratorTimeout.AsTimeSpan,
                 HttpClientType = typeof(ShardingStore),
+                DisableTopologyCache = true,
```

Comment:
DocumentConventions.DefaultForServer.DisableTopologyCache
also can you copy DisposeCertificate value as well? I think we need that

Reply:
Done